SparkPost/2web2ui
# Merge greenkeeper prs

This closes:
- all [`greenkeeper`](https://github.com/SparkPost/2web2ui/pulls?q=is%3Apr+is%3Aopen+label%3Agreenkeeper) prs those are currently passing ( except `mdx-loader` -  #682)
- #658 
- #680 
- #585 

Also covers:
- bumping npm requirements to `6.4.1`
- fixing HMR and error overlay
- remove eslint-plugin-flowtype package

Notes: 
- ~Report summary page showing `Warning: Failed prop type: Invalid prop scale supplied to XAxis` if I use latest version. This is confusing. Their doc says `utcTime` is valid value but in source i see it's `utc`. I've now restored to original version we had but still getting same warning.~ Removed the prop and default will be used by we provide custom ticks anyway. 
- mdx-loader needs to revisit. it's current state is confusing. seems library is renamed. but if I just update it to 2.0.0, it breaks UI (notification center)
- react-dropzone removed `disablePreview ` prop hence removed. But not sure why component name is renamed from `<d>` to `<n>`. 
- downshift renamed `default*` props to `initial*`. 

**How to test?**
- All tests pass w/o any warnings
- Random app uses
- Test some key ux including login, signup
- Hot module reloading works, react error overlay (trigger some syntax error to see)
- Production bundle creation works (Travis will verify this anyway)
- Review metrics/charts
